test/test_fspm.cpp

```diff
@@ -78,3 +83,44 @@ TEST_F (FspmUnitTest, FspmCheckValidateConfiguration)
    EXPECT_EQ (pf_fspm_validate_configuration (&cfg), -1);
    cfg.im_0_data.im_supported = 0;
 }
+
+TEST_F (FspmTest, FspmGetSystemLocation)
+{
+   pf_snmp_system_location_t first;
+   pf_snmp_system_location_t second;
+
+   memset (&first, '1', sizeof (first));
+   memset (&second, '2', sizeof (second));
+
+   pf_fspm_get_system_location (net, &first);
+   EXPECT_LT (strlen (first.string), sizeof (first.string));
```

Comment:
Isn't this true for any null terminated string, regardless of content?
`EXPECT_LT (strlen (first.string), sizeof (first.string));`

Reply:
Yes. The check tests that the returned string is null terminated.

Reply:
Ok, fine.